plugins/hiscore/init.lua

```diff
@@ -283,7 +309,11 @@ function hiscore.startplugin()
 		last_write_time = -10
 		emu.print_verbose("Starting " .. emu.gamename())
 		config_read = read_config();
-		local dat = read_hiscore_dat()
+		local dat = read_hiscore_dat("hiscore.dat")
+		if dat and dat == "" then
+			-- nothing found, try in console_hiscore.dat instead
+			dat = read_hiscore_dat("console_hiscore.dat")
+		end
```

Comment:
This seems very dubious.  The `hiscore.dat` file will always be present in a normal distribution.  With the code like this, the only way to trigger reading the `console_hiscore.dat` is to remove it.

Reply:
nope,  `read_hiscore_dat()` will return an empty string if no cluster/data lines are found for the current game. In this case `console_hiscore.dat` is also checked.